src/interface/desktop/chat.html

```diff
@@ -888,10 +888,10 @@
         }
 
         function renderFirstRunSetupMessage() {
-            first_run_message = `Hi 👋🏾, to get started:
+            first_run_message = `<p class="first_run_message_heading">Hi 👋🏾, to get started:<p>
             <ol>
-                <li>Generate an API token in the <a class='inline-chat-link' href="#" onclick="window.navigateAPI.navigateToWebSettings()">Khoj Web settings</a></li>
-                <li>Paste it into the API Key field in the <a class='inline-chat-link' href="#" onclick="window.navigateAPI.navigateToSettings()">Khoj Desktop settings</a></li>
+                <li class="inline-chat-text">Generate an API token <a class='inline-chat-link' href="#" onclick="window.navigateAPI.navigateToWebSettings()">Khoj Web settings</a></li>
```

Comment:
I was actually requesting that these classes be more specific to the first run message, such as below. Other than that, looks good! 🔥

```suggestion
                <li class="inline-chat-text">Generate an API token <a class='first-run-message-inline-link' href="#" onclick="window.navigateAPI.navigateToWebSettings()">Khoj Web settings</a></li>
```